ch05/03_bonus_pretraining_on_gutenberg/prepare_dataset.py

```diff
@@ -10,6 +10,12 @@
 import argparse
 import os
 import re
+from gutenberg.src.cleanup import strip_headers
+
+
+def is_english(text, threshold=0.9):
```

Comment:
I like how relatively simple this is. This will probably also result in many false negatives for some European languages, but that's totally ok, it doesn't need to be super sophisticated. This looks perfect for now.